lib/map.js

```diff
@@ -43,9 +44,13 @@ module.exports = function ({
 		const groundTruthsLabel = groundTruths.filter(({label}) => label === l).map(o => Object.assign({}, o, {match: false}));
 		const predictionsLabel = predictions.filter(({label}) => label === l).map(o => Object.assign({}, o, {match: false}));
 
-		return ap({groundTruthsLabel, predictionsLabel, iouThreshold});
+		return {label: l, ap: ap({groundTruthsLabel, predictionsLabel, iouThreshold})};
 	});
 
 	const l = aps.length;
-	return aps.reduce((a, b) => a + b, 0) / l;
+	const mAp = aps.reduce((a, b) => a + b.ap, 0) / l;
+	if (perLabelDetail) {
+		return {mAp, aPs: aps};
```

Comment:
let's use `mAP` instead of `mAp`

```suggestion
		return {mAP, aPs: aps};
```

Reply:
I guess you've changed your mind ? Because in the email notification I got, you suggested to use `mAp` and `aps` !?